lib/crypt_keeper/provider/postgres_pgp.rb

```diff
@@ -14,6 +15,8 @@ def initialize(options = {})
           raise ArgumentError, "Missing :key"
         end
 
+        @fallback_key = options.fetch(:fallback_key, nil)
```

Comment:
can you doc the fallback_key as an option?

Reply:
c98ca41